src/pages/workspace/accounting/PolicyAccountingPage.tsx

```diff
@@ -41,6 +48,103 @@ type PolicyAccountingPageProps = WithPolicyProps &
         policy: Policy;
     };
 
+const accountingIntegrations = Object.values(CONST.POLICY.CONNECTIONS.NAME);
+
+type AccountingIntegration = {
+    title: string;
+    icon: IconAsset;
+    setupConnectionButton: React.ReactNode;
+    connectedIntegrationMenuItems: MenuItemProps[];
+};
+function accountingIntegrationData(
+    connectionName: PolicyConnectionName,
+    policyID: string,
+    translate: LocaleContextProps['translate'],
+    styles: ThemeStyles,
+    isConnectedToIntegration?: boolean,
+    integrationToDisconnect?: PolicyConnectionName,
+): AccountingIntegration {
+    // eslint-disable-next-line default-case
+    switch (connectionName) {
+        case CONST.POLICY.CONNECTIONS.NAME.QBO:
+            return {
+                title: translate('workspace.accounting.qbo'),
+                icon: Expensicons.QBOSquare,
+                setupConnectionButton: (
+                    <ConnectToQuickbooksOnlineButton
+                        policyID={policyID}
+                        disconnectIntegrationBeforeConnecting={isConnectedToIntegration}
+                        integrationToDisconnect={integrationToDisconnect}
+                    />
+                ),
+                connectedIntegrationMenuItems: [
+                    {
+                        icon: Expensicons.Pencil,
+                        iconRight: Expensicons.ArrowRight,
+                        shouldShowRightIcon: true,
+                        title: translate('workspace.accounting.import'),
+                        wrapperStyle: [styles.sectionMenuItemTopDescription],
+                        onPress: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKS_ONLINE_IMPORT.getRoute(policyID)),
+                    },
+                    {
+                        icon: Expensicons.Send,
+                        iconRight: Expensicons.ArrowRight,
+                        shouldShowRightIcon: true,
+                        title: translate('workspace.accounting.export'),
+                        wrapperStyle: [styles.sectionMenuItemTopDescription],
+                        onPress: () => {},
+                    },
+                    {
+                        icon: Expensicons.Gear,
+                        iconRight: Expensicons.ArrowRight,
+                        shouldShowRightIcon: true,
+                        title: translate('workspace.accounting.advanced'),
+                        wrapperStyle: [styles.sectionMenuItemTopDescription],
+                        onPress: () => {},
+                    },
+                ],
+            };
```

Comment:
these and below items are almost the same, I think we should return just `onPress` method and define the rest once